### DISCUSSION TITLE
LDM Notes for October 19th, 2022

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6576] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-10-19.md

- Open questions in list patterns on `IEnumerable`
    - Allowing patterns after slices
    - Allowing slicing to capture

**Comment by kyoyama-kazusa** [comment]
Hi :) Can I ask a question not relative to the discussed syntax feature?

I wonder what does the part "Quote of the Day" really mean? 🤣 I found that all LDMs in recently months contain this part, but it seems that they are not really(?) relative to the LDM's topics respectively.

For example, this LDM has discussed the feature "List Patterns with IEnumerable Types", but "Quote of the Day" says something about children. 😄 Is there a relation between syntax feature and the quote? Am I missing something?

**Reply by CyrusNajmabadi** [reply]
Qotd are things started by varying members during the language design meeting. Usually things that caused a lot of laughs. They are intentionally included, without context, for maximum effect :-)

**Reply by kyoyama-kazusa** [reply]
Thank you :D
